src/components-examples/BUILD.bazel

```diff
@@ -97,16 +97,18 @@ package_docs_content(
 
         # For the live-examples in our docs, we want to package the highlighted files
         # into the docs content. These will be used to show the source code for examples.
-        ":highlighted-source-files": "examples-highlighted",
+        # Note: `examples-highlighted` is a tree artifact that we want to store as is
+        # in the docs-content. Hence there is no target section name.
+        ":examples-highlighted": "",
     },
     tags = ["docs-package"],
 )
 
 ng_package(
     name = "npm_package",
     srcs = ["package.json"],
-    data = [":docs-content"],
     entry_point = ":public-api.ts",
+    nested_packages = [":docs-content"],
```

Comment:
Can we add a comment that this is a workaround to store a tree artifact in the `ng_package`. `ng_package` does not properly handle tree artifacts currently so we escalate to `nested_packages`.

I can send an issue/fix upstream, but we should add a comment that we add here, and in the `package_docs_content` rule.

Reply:
Is the issue created yet? would be good to link it in the comment